FlightsWebsite/src/js/weather.js

```diff
@@ -7,6 +7,9 @@ function selectedWeather() {
         case 3:
             displayGraphTemperaturesAtJFK();
             break;
+        case 4:
```

Comment:
Are you sure that this is correct case for that chart? It is opening when I select "For three origins all temperatures attributes", not daily mean per origin.

Reply:
sorry .. my bad